src/FiscalYears/FiscalYearsList/FiscalYearsList.js

```diff
@@ -152,7 +153,11 @@ const FiscalYearsList = ({
 
       <Route
         path={`${FISCAL_YEAR_ROUTE}/:id/view`}
-        component={FiscalYearDetails}
+        render={() => (
+          <IfPermission perm="ui-finance.fiscal-year.view">
```

Comment:
is it required? as for me details is part of view 

just curious

Reply:
in case user use direct link without view permissions, it won't show details pane with labels and without any data